aiven/client/cli.py

```diff
@@ -3059,6 +3059,86 @@ def mirrormaker__replication_flow__delete(self):
             self.args.target_cluster,
         )
 
+    @arg.project
+    @arg.service_name
+    def service__flink__table__list(self):
+        """List created Flink tables"""
```

Comment:
```suggestion
        """List Flink tables"""
```

created seems extra word here